packages/react/src/context.js

```diff
@@ -19,16 +19,27 @@ let EmotionCacheContext: React.Context<EmotionCache | null> = /* #__PURE__ */ Re
 
 export let CacheProvider = EmotionCacheContext.Provider
 
-let withEmotionCache = function withEmotionCache<Props, Ref: React.Ref<*>>(
-  func: (props: Props, cache: EmotionCache, ref: Ref) => React.Node
-): React.AbstractComponent<Props> {
+let withEmotionCache = function withEmotionCache<
```

Comment:
I don't think changes here are required - `withEmotionCache` is purely internal and you have already covered all cases that use it.

Reply:
Fixed in 5227c72